html/js/melpa.js

```diff
@@ -150,7 +151,7 @@
       pkgs.push(new melpa.Package({
         name: name,
         version: version,
-        commit: built.props.commit,
+        commit: props.commit || '',
```

Comment:
Was this change actually necessary? It looks undesirable to me, since it will make every package look like it has commit info.